src/nunit.analyzers.tests/ClassicModelAssertUsage/ClassicModelAssertUsageAnalyzerTests.cs

```diff
@@ -569,6 +573,38 @@ public void Test()
         {
             ↓ClassicAssert.IsNotInstanceOf<int>(2);
         }
+    }");
+            RoslynAssert.Diagnostics(this.analyzer, expectedDiagnostic, testCode);
+        }
+
+        [Test]
+        public void AnalyzeWhenPositiveIsUsed()
+        {
+            var expectedDiagnostic = ExpectedDiagnostic.Create(AnalyzerIdentifiers.PositiveUsage);
+
+            var testCode = TestUtility.WrapClassInNamespaceAndAddUsing(@"
+    public sealed class AnalyzeWhenPositiveIsUsed
+    {
+        public void Test()
+        {
+            ↓ClassicAssert.Positive(2);
+        }
+    }");
+            RoslynAssert.Diagnostics(this.analyzer, expectedDiagnostic, testCode);
+        }
+
+        [Test]
+        public void AnalyzeWhenNegativeIsUsed()
+        {
+            var expectedDiagnostic = ExpectedDiagnostic.Create(AnalyzerIdentifiers.NegativeUsage);
+
+            var testCode = TestUtility.WrapClassInNamespaceAndAddUsing(@"
+    public sealed class AnalyzeWhenNegativeIsUsed
+    {
+        public void Test()
+        {
+            ↓ClassicAssert.Negative(2);
+        }
     }");
```

Comment:
Should we also add a method for `IsAssignableFromUsage` and `IsNotAssignableFromUsage`

Reply:
Thanks yes, I must have been rushing it that I forgot. Added.